bindata/kube-proxy/kube-proxy.yaml

```diff
@@ -161,6 +161,9 @@ spec:
       - operator: Exists
       nodeSelector:
         kubernetes.io/os: linux
+{{ if eq .OVN_DPU_MODE "dpu" }}
+        network.operator.openshift.io/dpu: ''
+{{ end }}
```

Comment:
Maybe

```
{{- if .KUBE_PROXY_NODE_SELECTOR }}
        {{ .KUBE_PROXY_NODE_SELECTOR }}
{{- end }}
```

and update the go code accordingly? Rather than putting DPU-specific stuff in this generic template...

Reply:
Done.